AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -648,6 +648,7 @@ reddit.com#@#.ADBAR
 ! SECTION: Anti-adblock - Regular rules
 ! Do not forget about '!#safari_cb_affinity' section for exclusions
 !
+||alias-1.azpelis.com/dectector.js
```

Comment:
It cannot be closed, so for Base filter.
Also not covered another subdomains and the main domain https://azpelis.com/

Reply:
Okay.